templates/master/00-master/_base/files/usr-local-bin-etcd-snapshot-restore-sh.yaml

```diff
@@ -25,6 +25,11 @@ contents:
         usage
     fi
 
+    if [ ! -f "$1" ]; then
+      echo "Backup file $1 file does not existBackup file. Please try again "
```

Comment:
can we clarify this message a little?

`etcd snapshot $SNAPSHOT_FILE does not exist.`

Reply:
+1

Reply:
since we capture $1 as SNAPSHOT_FILE I think it more readable to check that in you if block. WDYT?

Reply:
+1 I think it will be more readable. I will move the check after the variables are defined.